.github/workflows/build.yml

```diff
@@ -0,0 +1,95 @@
+name: Dolphin-AppImage
+
+on: 
+  push:
+
+jobs:
+  dolphin-appimage:
+    name: dolphin bundled as appimage
+    runs-on: ubuntu-latest
```

Comment:
If you use ubuntu-latest (and so a quite recent libc), it basically defeats the purpose of appimage (run on any "as old as possible OS"). Suggestion: 16.04, 18.04 or Debian 9.

Reply:
Yes, you are correct. On my version I run ubuntu-18.04 ( LTS ).